### PR TITLE
tutorial docs: Fix links and add extra notes to remove ambiguity. 

### PR DESCRIPTION
**Pull request by YJDave**
This PR fixes links and modifies/add some content in tutorial.

### CODE REVIEW

File: docs/tutorial_step1.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Now run Sugarizer, you should see the icon of a new activity like this:
 
 ![](images/tutorial_step1_2.png)
 
+Note: If you are not able to see the icon of newly added activity, empty the browser cache and reload
```
Comment (llaske) [inline docs/tutorial_step1.md]: Not sure why it's needed.

File: docs/tutorial_step4.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ So we will update our request on environment in `activity/activity.js` to test t
 		}
 	});
 
-Let's display the JavaScript console on your browser and test it. Launch the Pawn activity from the **Start new** menu or from List view of activities. Here is the result:
+Let's display the JavaScript console on your browser and test it. Launch the Pawn activity from the **Start new** menu(hover mouse over activity icon, list will appear, click on **Start new**) or from List view of activities. Here is the result:
```
Comment (llaske) [inline docs/tutorial_step4.md]: We don't need this precision. It's a standard behavior in Sugarizer.

File: docs/tutorial_step6.md [hunk 3]
```diff
@@ -41,7 +45,7 @@ Then, once the activity will be open, he will share the activity by clicking on
 
 ![](images/tutorial_step6_5.png)
 
-From your Neighborhood view you will see the icon of the activity suddenly appear near the Michaël's icon. Just pass the mouse on the activity icon and click the Join menu and you will be able to join Michaël.
+When you go to Neighborhood view(click on **Neighborhood view** icon from homepage as shown previously) from your(LIONEL's) browser, you will see the icon of the activity suddenly appear near the Michaël's icon. Just pass the mouse on the activity icon and click the Join menu and you will be able to join Michaël.
```
Comment (llaske) [inline docs/tutorial_step6.md]: It seems clear in the current paragraph.

File: docs/tutorial_step1.md (hunk 1 above)
Comment (YJDave) [reply docs/tutorial_step1.md]: In some browser(in my case Safari) if you don't hard reload(removing cache) it doesn't reflect new changes. I myself encounter this error, and thought that small note might help other beginners.

File: docs/tutorial_step4.md (hunk 2 above)
Comment (YJDave) [reply docs/tutorial_step4.md]: Okay, make sense.

File: docs/tutorial_step6.md (hunk 3 above)
Comment (YJDave) [reply docs/tutorial_step6.md]: In tutorial `Neighborhood view` is referred to both of following icon
* `Neighborhood view` button from homepage
* `Neighborhood view` button from any particular activity page

This make so confusing that tutorial is suggesting to join `Neighborhood view` from activity page from one user client and to open `Neighborhood view` from homepage from another user client. So I though adding some more clarification would be helpful.

**Comment by YJDave** [comment]
@llaske Thank you for detailed review!
I have addressed all your comments, please let me know your views. I have also updated PR to remove the changes in `tutorial Step 4: Explain how to start new activity.` which I agree that isn't necessarily required.

**Comment by llaske** [comment]
This tutorial is not a Sugarizer tutorial but a tutorial to learn how to develop an activity in Sugarizer.
So it should not explain step by step the Sugarizer UI. I'm not agree to add notes that are not useful for this initial objective.
Plus, no user - including young people from Google Code In - tell us that the current description is ambiguous.

**Comment by YJDave** [comment]
@llaske Thanks for your response! 

> This tutorial is not a Sugarizer tutorial but a tutorial to learn how to develop an activity in Sugarizer.
So it should not explain step by step the Sugarizer UI.

Didn't thought this way. I will close the PR then, as these changes are not required.

**Pull request closed without merging**